.github/CHANGELOG.md

```diff
@@ -186,6 +186,43 @@
   engine = sf.RemoteEngine("X8", connection=connection)
   ```
 
+* The `sf.api.Job` class has been replaced with the
+  [xcc.Job](https://xanadu-cloud-client.readthedocs.io/en/stable/api/xcc.Connection.html)
+  class.
+  [(#645)](https://github.com/XanaduAI/strawberryfields/pull/645)
+
+  A `Job` object is returned when running jobs asynchronously. In previous versions of Strawberry
+  Fields (v0.19.0 and lower), the `Job` object can be used as follows:
+
+  ```pycon
+  >>> job = engine.run_async(program, shots=1)
+  >>> job.status
+  "queued"
```

Comment:
Minor point, but I _think_ the Python interpreter outputs strings in single quotes.
```suggestion
  'queued'
```
Same goes for the rest below.